rln/Cargo.toml

```diff
@@ -3,76 +3,39 @@ name = "rln"
 version = "0.1.0"
```

Comment:
@richard-ramos @staheri14  of the changes in the `rln` crate, which of these are must haves from your POV? Both for normal integration (e.g. bugs found in nwaku and go-waku) as well as for the experimental wasm support?

Trying to figure out which things should be part of initial v0.1 release and which are OK to deal with later.